cmd/eksctl-anywhere/cmd/listpackages.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"context"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+)
+
+type listPackagesOption struct {
+	kubeVersion string
+	source      curatedpackages.BundleSource
+}
+
+var lpo = &listPackagesOption{}
+
+func init() {
+	listCmd.AddCommand(listPackagesCommand)
+	listPackagesCommand.Flags().Var(&lpo.source, "source", "Discovery Location. Options (cluster, registry)")
+	listPackagesCommand.Flags().StringVar(&lpo.kubeVersion, "kubeversion", "", "Kubernetes Version of the cluster to be used. Format <major>.<minor>")
+	err := listPackagesCommand.MarkFlagRequired("source")
+	if err != nil {
+		log.Fatalf("Error marking flag as required: %v", err)
+	}
+}
+
+var listPackagesCommand = &cobra.Command{
+	Use:          "packages",
+	Short:        "Lists curated packages available to install",
+	PreRunE:      preRunPackages,
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if err := validateKubeVersion(lpo.kubeVersion, lpo.source); err != nil {
```

Comment:
Should we validate too that `source` has a valid value?

Reply:
The validation occurs in BundleSource.